webviz_subsurface/ert_jobs/smry2arrow.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+"""Convert UNSMRY file to Apache Arrow IPC file format (also known as Feather V2)
+"""
+
+from typing import List, Iterable, Dict
+import argparse
+import re
+from pathlib import Path
+import logging
+import json
+import time
+
+from ecl.summary import EclSum, EclSumKeyWordVector
+import pyarrow as pa
+import pyarrow.feather as feather
+
+logger = logging.getLogger(__name__)
+
+
+DESCRIPTION: str = """
+Convert UNSMRY file to Apache Arrow IPC file format (also known as Feather V2)
+"""
+CATEGORY: str = "utility.eclipse"
+EXAMPLES: str = """
+Convert and output summary data in SOME.UNSMRY file to an Arrow IPC file named unsmry.arrow
+this in the ert workflow:
+
+    FORWARD_MODEL SMRY2ARROW(<INPUT>=eclipse/model/SOME.UNSMRY, <OUTPUT>=share/results/tables/unsmry.arrow)
+
+"""  # noqa
+
+
+def _get_parser() -> argparse.ArgumentParser:
+    """Setup parser for command line options"""
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description=DESCRIPTION,
+    )
+    parser.add_argument(
+        "input",
+        type=Path,
+        help="Input file",
+    )
+    parser.add_argument(
+        "output",
+        type=Path,
+        help="Output file",
+        default=Path() / "share" / "results" / "tables" / "unsmry.arrow",
+    )
+    return parser
+
+
+# TODO(Sigurd) This function should be shared with export_connection_status.py
```

Comment:
Make GitHub issue or within the scope of this PR?

Reply:
Created new issue: https://github.com/equinor/webviz-subsurface/issues/692
Will update the comment